packages/input-amount/src/formatters.js

```diff
@@ -23,3 +23,10 @@ export function formatAmount(modelValue, givenOptions) {
 
   return formatNumber(modelValue, options);
 }
+
+export function normalizeCurrencyLabel(currency, locale) {
```

Comment:
Good fix. Maybe it's an idea to put this function in the `localize` package as a file inside `src/number`? There we keep all corrections on top of `Intl.Numberformat`

Reply:
@tlouisse Thanks, exactly did that. Is it good now ?